.azure/pipelines/ci.yml

```diff
@@ -67,6 +69,8 @@ variables:
              /p:OfficialBuildId=$(Build.BuildNumber)
     - name: _SignType
       value: real
+    - name: _TestBuildSkipArgs
+      value: /p:SkipTestBuild=true
```

Comment:
Why is this inconsistent? Suggest we use `/p:SkipTestBuild=true` unconditionally in the Build&hellip; jobs

Reply:
I left it in for PRs because I wasn't sure that all the tests got properly built in the other jobs. If that's the case then I can enable it for all build jobs.